andi/templates/secrets.yaml

```diff
@@ -15,4 +15,12 @@ metadata:
   name: {{ .Release.Name }}-andi-auth0-client-secret
 type: Opaque
 stringData:
-  auth0_client_secret: {{ quote .Values.auth.auth0_client_secret }}
\ No newline at end of file
+  auth0_client_secret: {{ quote .Values.auth.auth0_client_secret }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-andi-aws-access-key-secret
+type: Opaque
+stringData:
+  aws_access_key_secret: {{ quote .Values.aws.accessKeySecret }}
```

Comment:
Could both the key ID and the key secret be stored in the secret so that this secret acts as a whole set of AWS credentials?

Reply:
We definitely can do this! I guess my question would be what additional benefit this would add? My understanding is that the AWS Access Key ID is not sensitive, which is why we didn't store it as part of the secret.

Reply:
The benefit I perceive is to readability by keeping the AWS access credentials together as one logical unit.  It leaves us in a better position for making structural changes in the future

Reply:
@ksmith-accenture - sounds like a plan! This change has been made. We're ready for you to re-review whenever you have chance!